docs/reference/elasticsearch/jvm-settings.md

```diff
@@ -139,9 +139,25 @@ If you are running {{es}} as a Windows service, you can change the heap size usi
 
 ## JVM heap dump path setting [heap-dump-path-setting]
 
-By default, {{es}} configures the JVM to dump the heap on out of memory exceptions to the default logs directory. On [RPM](docs-content://deploy-manage/deploy/self-managed/install-elasticsearch-with-rpm.md) and [Debian](docs-content://deploy-manage/deploy/self-managed/install-elasticsearch-with-debian-package.md) packages, the logs directory is `/var/log/elasticsearch`. On [Linux and MacOS](docs-content://deploy-manage/deploy/self-managed/install-elasticsearch-from-archive-on-linux-macos.md) and [Windows](docs-content://deploy-manage/deploy/self-managed/install-elasticsearch-with-zip-on-windows.md) distributions, the `logs` directory is located under the root of the {{es}} installation.
+Depending on your stack version, {{es}} configures the JVM to dump the heap on out of memory exceptions to the following location by default:
+
+* {applies_to}`stack: ga 9.1` The default logs directory 
```

Comment:
This should also include 8.19?

Reply:
this docs set only applies to 9.0+, but I agree that it is confusing. let me see if there's something I'm able to do.

Reply:
verified that we need to keep the 8.x and 9.x docs separate - this info should be manually backported into the asciidoc docs in the 8.19 branch. 

will likely be doing a validation pass that the changes that are making it into 8.19 were appropriately ported but haven't gotten there yet :)